server/server.js

```diff
@@ -135,8 +163,29 @@ var upload = multer({
  */
 var exonFile = bindingPath + 'exons.bin';
 
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(bodyParser.json());
+/**
+ * User authenticate handler.
+ */
+MongoClient.connect(mongoUrl, function(err, db) {
+  if (err) {
+    log.serverLog(err.message);
+    return;
+  }
+  log.serverLog('connected to MongoDB');
+  database.db = db;
+
+  // Start the application.
+  //var privateKey = fs.readFileSync(privateKeyPath);
```

Comment:
It is still not over https. You may add a separate PR to address, and mark TODO. https is essential and should be enforced ASAP. @Shauro @djasmine